data_model/src/query/mod.rs

```diff
@@ -268,10 +261,16 @@ impl PartialOrd for TransactionQueryOutput {
 impl Ord for TransactionQueryOutput {
```

Comment:
sorry, I just noticed this is inconsistent. Remove `PartialOrd` and `Ord` manual impls and derive them